README.md

```diff
@@ -64,17 +65,18 @@ community.
    is welcome to participate in the review.
 
 3. If there are many uncertainties about the proposal, consider first publishing
-   document somewhere else and discussing it via forum or e-mails. That will
-   greatly reduce amount of back-and-forth changes in the DIP pull request later.
+   document somewhere else and discussing it via the [NG](http://forum.dlang.org/group/general)
+   or e-mails. That will greatly reduce amount of back-and-forth changes in the
+   DIP pull request later.
 
-## DIPs by language authors
+## DIPs by the D language authors
```

Comment:
Should we define somewhere that the language authors are A&W by definition?
Otherwise we could also use a term like "Presidents of the D Language Foundation" or "Officers of the D Language Foundation"